Examples/Python/left_atrium.py

```diff
@@ -68,8 +68,8 @@ def Run_Pipeline(args):
         fileList_seg = sorted(glob.glob(parentDir + "segmentations/*.nrrd"))
 
     if args.tiny_test:
-        fileList_img = fileList_img[:2]
-        fileList_seg = fileList_seg[:2]
+        fileList_img = fileList_img[:5]
```

Comment:
I like that these extra images are being added, but perhaps we should include a comment about why (because the first two images are the same resolution so the issue being addressed by this pull request only comes up when images are diff res are included).

Reply:
Done